src/lib/CMakeLists.txt

```diff
@@ -663,7 +663,7 @@ set_source_files_properties(
 # Add our jemalloc build as an interface library so that the build process of hyrise does not have to wait for it
 # -rdynamic tells the linker to export the library's symbols so that plugins can use them.
 target_link_libraries(hyrise_impl PUBLIC ${LIBRARIES} INTERFACE custom_jemalloc -rdynamic)
-target_link_libraries_system_private(hyrise_impl nlohmann_json::nlohmann_json)
```

Comment:
I don't get what this change from private linking to public linking for `nlohmann_json` does. The description does not help me. The removed line with `private` means that other components linking to `hyrise_impl` do nonetheless not see the linked `nlohmann_json`?

Reply:
Note that the line was not removed, it was just changed from private to public (see my last reply)

Reply:
Yeah I know, I think was suprised me was that you can “hide” inclusion/libs for “transitive” includes.